master/internal/rm/kubernetesrm/pods.go

```diff
@@ -399,7 +401,11 @@ func (p *pods) startClientSet() error {
 		return errors.Wrap(err, "failed to initialize kubernetes clientSet")
 	}
 
-	for _, ns := range append(maps.Keys(p.namespaceToPoolName), p.namespace) {
```

Comment:
Now that i'm reading the code, it is really a shame to break this because it's so simple to not break.

Just triple checking @hkumar92, you made sure no one relies on this right? At some point I had conviction but now I'm conflicted. I think you are right that workspace -> namespace mapping was right all along (and, in general, want platform features to be orthogonal/obvious to use so I don't want both) but I don't want to break an OSS user who is using RP -> namespace mappings.

wdyt?

Reply:
I checked with CE team here: https://hpe-aiatscale.slack.com/archives/C030AQB5D8E/p1710876793956859. I am triple checking there as well but operating assumption is yes we should break that, call it out and provide a mechanism for them to link workspaces to namespaces in the CLI before we break it